compose/common.yml

```diff
@@ -188,10 +191,17 @@ services:
     restart: on-failure
 
 secrets:
+  # Replicates the db-sync one for historical reasons.
```

Comment:
This comment is a bit confusing. Let's discuss on call and I can help refine.

Reply:
Thanks for the chat @iccicci. OK how about using this as content, formatted as you please:

The upstream Docker image is missing environment variables that allow the consumer to mount into a non-standard path, so we must conform with the hard-coded secret name. As demonstrated here: https://docs.docker.com/compose/use-secrets/#advanced, we could extend the Docker image and define POSTGRES_DB_FILE as the custom path to remove this workaround. See https://github.com/input-output-hk/cardano-db-sync/blob/aed18d1be192bd58d054cdba23758b579dae9f4e/nix/docker.nix#L116

Reply:
cba27e3cf should solve this